### PR TITLE
Book class barebones functionality and testing setters

### PR DESCRIPTION
**Pull request by nickdchee**

### CODE REVIEW

File: app/src/main/java/comnickdchee/github/a3am/Book.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+import android.media.Image;
+
+public class Book {
+    private String ISBN;
+    private String title;
+    private String author;
+    private Image image;
+    //private User Owner;
+    private enum STATUS {BORROWED, AVAILABLE};
```
Comment (ismaeelBM) [inline app/src/main/java/comnickdchee/github/a3am/Book.java]: Statuses Available, Requested, Accepted, Borrowed

File: app/src/main/java/comnickdchee/github/a3am/Book.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    //private requests ArrayList<User>;
+    //private User currentBorrower;
```
Comment (ismaeelBM) [inline app/src/main/java/comnickdchee/github/a3am/Book.java]: Uncomment all User things and write it we will merge it later.

File: app/src/test/java/comnickdchee/github/a3am/ExampleUnitTest.java [hunk 3]
```diff
@@ -14,4 +17,39 @@
...
+        }
+    }
+    @Test
+    public void getTitle() {
+        {
+            assertEquals(b.getTitle(), "HarryPotter");
+        }
+    }
+    @Test
+    public void setAuthorName() {
```
Comment (ismaeelBM) [inline app/src/test/java/comnickdchee/github/a3am/ExampleUnitTest.java]: Rename these test functions using _test_methodName_ as your signature. Also, have each test create a new instance of the Book class.

**Comment by nickdchee** [comment]
I'll be responsible for resolving these

**Comment by nickdchee** [comment]
I'm closing this pull request. Once Ismaeel is done with the unit tests in my branch, I want to merge both branches and check that all the conflicts are resolved.

**Pull request closed without merging**